### PR TITLE
finalised scatter plot

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

**Comment by maxwellfundi** [comment]
@dannyparsons  here is the finalised scatterplot now

File: instat/dlgScatterPlot.vb [hunk 1]
```diff
@@ -1,3 +1,104 @@
...
+        'tests when okay is enable
+        If ucrReceiverX.IsEmpty() = True Then
+            ucrBase.OKEnabled(False)
+            ucrBase.clsRsyntax.RemoveParameter("x")
+        ElseIf ucrReceiverY.IsEmpty() = True Then
+            ucrBase.clsRsyntax.RemoveParameter("y")
+            ucrBase.OKEnabled(False)
+        Else
+            ucrBase.OKEnabled(True)
+        End If
```
Comment (dannyparsons) [inline instat/dlgScatterPlot.vb]: Same idea as my comment on line plot https://github.com/africanmathsinitiative/Instat/pull/903/files#r54332401

**Comment by dannyparsons** [comment]
Looks great, just one comment to look at.

**Comment by maxwellfundi** [comment]
@dannyparsons  this is now corrected

**Pull request merged**